### PR TITLE
RS/JX/Rule16-6

### PR DESCRIPTION
**Pull request by Jiarongx-Xie**
This rule has  been tested.

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section16/section16rule6.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            description="When included in the proposed design, the baseline elevator cab ventilation fan power shall be 0.33 W/cfm.",
+            ruleset_section_title="Elevators",
+            standard_section="Table G3.1-16 Baseline Building Performance",
+            is_primary_rule=True,
+            list_path="ruleset_model_descriptions[0].buildings[*].elevators[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmd_p = context.PROPOSED
+        return find_all("$.ruleset_model_descriptions[0].buildings[*].elevators", rmd_p)
```
Comment (yunjoonjung-PNNL) [inline rct229/rulesets/ashrae9012019/section16/section16rule6.py:34]: Can you add [*] at the end of the json path?

**Review by yunjoonjung-PNNL — APPROVED** [review]

**Review by weilixu — COMMENTED** [review]
This PR has the same issue as the other section 16 rules - if the baseline has no elevator, this rule evaluation will not return any outcome, thus result in pass.

File: rct229/rulesets/ashrae9012019/section16/section16rule6.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            description="When included in the proposed design, the baseline elevator cab ventilation fan power shall be 0.33 W/cfm.",
+            ruleset_section_title="Elevators",
+            standard_section="Table G3.1-16 Baseline Building Performance",
+            is_primary_rule=True,
+            list_path="ruleset_model_descriptions[0].buildings[*].elevators[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmd_p = context.PROPOSED
+        return find_all(
```
Comment (yunjoonjung-PNNL) [inline rct229/rulesets/ashrae9012019/section16/section16rule6.py:34]: Could you follow [this comment](https://github.com/pnnl/ruleset-checking-tool/pull/1342#:~:text=%40JacksonJ%2DKC%20%2C%20could%20you%20check%20what%20outcome%20should%20it%20be%20if%20the%20baseline%20has%20no%20elevator%20but%20proposed%20has%3F%0AThe%20rule%20passes%20if%20no%20elevator%20in%20baseline%20model%20but%20proposed%20has.) and fix this is applicable logic?

**Comment by weilixu** [comment]
@Jiarongx-Xie  I added `assert_` to make sure both `power` and `flow` are non zero.
The logic here is if power or flow are 0, then it must be data error.

**Pull request merged**